nomad/resource_job_test.go

```diff
@@ -3272,6 +3328,37 @@ job "foo-multiregion" {
 }
 `
 
+var testResourceJobScheduleBlock = `
```

Comment:
hey neat!  if you're adding extra testing for this, do we also want to cover the new [`ui{}` block](https://developer.hashicorp.com/nomad/docs/job-specification/ui)?

`raw_exec` also has the new [cgroup overrides](https://developer.hashicorp.com/nomad/docs/drivers/raw_exec#cgroup_v1_override)...

how much do we test that kind of thing here in the TF provider?

Reply:
I guess the raw_exec cgroup stuff is part of the driver config, which probably(?) doesn't get parsed here(?)

Reply:
I usually add a test which at least covers the version, but the UI makes sense to add, so this can be found in: https://github.com/hashicorp/terraform-provider-nomad/pull/462/commits/67d75b858ab972c08d1b6d81646a258eb30a2986